scripts/tags/exturl.js

```diff
@@ -0,0 +1,57 @@
+/* global hexo */
+// Usage: {% exturl text url "title" %}
+// Alias: {% extlink text url "title" %}
+
+'use strict';
+
+var util = require('hexo-util');
```

Comment:
@ivan-nginx It seems the `hexo-util` module resolution would be failed if user didn't install it via `npm install` command. That is to say, it is required to install `hexo-util` module in order to use the `exturl` tag?  Issue #1490